documentation/AuthServer.md

```diff
@@ -25,56 +53,110 @@ The document types used by the authorization server include:
+| `type` | String | The document type for storing GPII tokens. | The value must be set to "onboardedSolutionAuthorization". |
+| `gpiiToken` | String | The GPII token that the authorization is associated with. | None |
+| `clientId` | String | The client id that the authorization is associated with. | None |
 | `selectedPreferences` | Object | The preferences that the user has granted permissions to the client to retrieve. | None |
-| `revoked` | Boolean | Whether this decision has been revoked. | false |
+| `revoked` | Boolean | Whether this authorization has been revoked. | false |
 
-#### Authorization Codes
+#### Privileged Prefs Creator Clients
 
 | Option | Type | Description | Default |
 | ------ | ---- | ----------- | ------- |
-| `id` | String | The authorization code id. Can be a UUID or any unique string. This value is saved into `_id` field in CouchDB/PouchDB. | None |
-| `type` | String | The document type for storing GPII tokens. | The value must be set to "authCode". |
-| `authDecisionId` | String | The authorization decision that this code is associated with. | None |
-| `code` | String | The intermediary code granted by the authorization server to identify the client. See [The OAuth 2.0 Authorization Framework](https://tools.ietf.org/html/rfc6749#section-1.3.1) for details. | None |
+| `id` | String | The client id. Can be a UUID or any unique string. This value is saved into `_id` field in CouchDB/PouchDB. | None |
+| `type` | String | The document type for storing the client type information. | The value must be set to "privilegedPrefsCreatorClient". |
+| `name` | String | The client name. | None |
+| `oauth2ClientId` | String | The unique identifier issued to a registered OAuth2 client by the authorization server. | None |
+| `oauth2ClientSecret` | String | Confidential shared secret between the client and the authorization server, used to verify the identity of the client. | None |
 
-#### Client Credentials Tokens
+#### Privileged Prefs Creator Authorizations
 
 | Option | Type | Description | Default |
 | ------ | ---- | ----------- | ------- |
-| `id` | String | The client credentials token id. Can be a UUID or any unique string. This value is saved into `_id` field in CouchDB/PouchDB. | None |
-| `type` | String | The document type for storing client credentials tokens. | The value must be set to "clientCredentialsToken". |
+| `id` | String | The privileged prefs creator id. Can be a UUID or any unique string. This value is saved into `_id` field in CouchDB/PouchDB. | None |
+| `type` | String | The document type for storing privileged prefs creators. | The value must be set to "privilegedPrefsCreatorAuthorization". |
 | `clientId` | String | The client id that this token is assigned to. | None |
 | `accessToken` | String | The access token used to retrieved the protected user preferences. | None |

```

Comment:
For Privileged Prefs Creator Authorizations, the `accessToken` indicates authorization to add preferences, rather than retrieve.